src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -93,23 +93,48 @@ fi
 
 cd "$SOURCE_DIR"
 
-echo
-echo "Running '{{ PackageInstallCommand }}'..."
-echo
-{{ PackageInstallCommand }}
-
 {{ if CustomRunBuildCommand | IsNotBlank }}
```

Comment:
Move this `CustomRunBuildCommand` into the `else` section as having this separate section feels repetitive.
For reference, before your changes in this PR, this is how it was looking like:

![image](https://user-images.githubusercontent.com/6559784/95115329-2490eb80-06fa-11eb-8be9-6350521e0f34.png)

Reply:
I agree it looks repetitive because we added this `packageInstallCommand `at multiple places, however since `CustomRunBuildCommand `has priority over other commands, we'll need to put it in the first `if` section.